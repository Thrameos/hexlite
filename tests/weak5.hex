p(a).
p(b).
p(c).

q(x).
q(y).

% 6 instances, but only 2 different values for Y, therefore costs should be 2@1
:~ p(X), q(Y). [1@1,Y]
