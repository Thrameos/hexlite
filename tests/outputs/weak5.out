{p(a),p(b),p(c),q(x),q(y)} <[2:1]>
